app/models/kanji.rb

```diff
@@ -1,6 +1,12 @@
 class Kanji < ApplicationRecord
-  validates :strokes, presence: true
-  validates :character, presence: true, length: { is: 1 }
+  has_many :cards, dependent: :destroy
```

Comment:
Interesting. I'm not sure our users would be happy if their cards disappeared.

Reply:
What about when kanji itself is deleted from the database? Will we keep cards which have that kanji or our assumption is that kanji will not be deleted?

Reply:
Honestly, I'm not sure how that should work on a technical level, but I wouldn't want the user to lose their picture data.